src/lib.rs

```diff
@@ -274,6 +274,33 @@ impl Element {
         self.children.iter_mut().find(predicate)
     }
 
+    /// Transverse element using an xpath-like string: root/child/a
+    pub fn find(&self, path: &str) -> Option<&Element> {
+        Self::find_path(&path.split('/').collect::<Vec<&str>>(), &self)
+    }
+
+    pub fn find_text(&self, path: &str) -> &str {
+        self.find(path)
+            .expect(&format!("Could not find {:?}", path))
+            .text.as_ref()
+            .expect(&format!("Path {:?} to have text", path))
+    }
+
+    pub fn find_int(&self, path: &str) -> i32 {
```

Comment:
I think it would be helpful to have a generic convenience method `find_value<T>` which returns `Result<Option<T>, E>` as above. It should work as long as we constrain to `T: FromStr`. This way, we'll support all of rust's primitives, and any lib/user types that implement `FromStr`.

I'm thinking of having such that one can call:
```
let n: u32 = doc.find_value("foo/bar/baz")?.unwrap_or(0);
let s: f64 = doc.find_value("foo/bar/boo")?.unwrap_or(3.14);
```